proposed/http-client/http-client.md

```diff
@@ -62,8 +61,8 @@ interface ClientInterface
      * Sends a PSR-7 request and returns a PSR-7 response. 
      * 
      * Every technically correct HTTP response MUST be returned as is, even if it represents a HTTP 
-     * error response or a redirect instruction. The only exception is 1xx responses which MUST be 
-     * assembled in the HTTP client. 
+     * error response or a redirect instruction. The only special case is 1xx responses, which MUST
+     * be assembled in the HTTP client.
```

Comment:
i actually don't like the word exception here, as its not about throwing an Exception... (thinking about non-native speakers who might get confused by the wording). does it make sense like this? both grammatically and semantically?